### PR TITLE
Hotfix master - Fix trailing-semicolon issue with nested properties

### PR DESCRIPTION
**Pull request by bgriffith**
Fixes the trailing-semicolon false positive highlighted in #352 

DCO 1.1 Signed-off-by: Ben Griffith gt11687@gmail.com

### CODE REVIEW

File: tests/sass/trailing-semicolon.scss [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
-  content: 'baz', 'fail'
+  padding: 0
 
   .qux {
     content: 'waldo'
   }
 }
+
+.foo {
+  font: {
```
Comment (DanPurdy) [inline tests/sass/trailing-semicolon.scss]: Can we add some test cases where it should fail here just to make sure?

**Comment by bgriffith** [comment]
Yeah good shout.

**Comment by DanPurdy** [comment]
also try this against it

```scss
.test {
    text: {
        line: {
            through: {
                color: red;
            }
        }
        orientation: landscape;
    }
}
```

**Comment by bgriffith** [comment]
Will do. I think it still needs some work. I'm having a issue.

**Comment by bgriffith** [comment]
Ok added more test cases and also added tests to test for non-default options (include: false)

**Comment by DanPurdy** [comment]
Update me and we're good to go

**Pull request merged**